src/python/dxpy/utils/completer.py

```diff
@@ -416,28 +416,47 @@ class InstanceTypesCompleter():
 
     # AWS
     aws_preferred_instance_types = OrderedDict()
-    for i in (InstanceTypeSpec('mem1_ssd1_v2_x2', 4.0, 47, 2),
-              InstanceTypeSpec('mem1_ssd1_v2_x4', 8.0, 93, 4),
-              InstanceTypeSpec('mem1_ssd1_v2_x8', 16.0, 186, 8),
-              InstanceTypeSpec('mem1_ssd1_v2_x16', 32.0, 372, 16),
-              InstanceTypeSpec('mem1_ssd1_x32', 60.0, 640, 32),
-              InstanceTypeSpec('mem1_ssd1_v2_x36', 72.0, 837, 36),
-
-              InstanceTypeSpec('mem2_ssd1_v2_x2', 8.0, 69, 2),
-              InstanceTypeSpec('mem2_ssd1_v2_x4', 16.0, 139, 4),
-              InstanceTypeSpec('mem2_ssd1_v2_x8', 32.0, 279, 8),
-
-              InstanceTypeSpec('mem3_ssd1_v2_x2', 16.0, 69, 2),
-              InstanceTypeSpec('mem3_ssd1_v2_x4', 32.0, 139, 4),
-              InstanceTypeSpec('mem3_ssd1_v2_x8', 64.0, 279, 8),
-              InstanceTypeSpec('mem3_ssd1_v2_x16', 128.0, 558, 16),
-              InstanceTypeSpec('mem3_ssd1_v2_x32', 256.0, 1166, 32),
+    for i in (InstanceTypeSpec('mem1_ssd1_v2_x2', 4.0, 50, 2),
```

Comment:
The previous storage numbers are accurate, they are the available space on the formatted disk.

Reply:
rollback 32.  Keep the round number to make consistent with the documentation.